### ISSUE TITLE
nanotest is not working on windows/jruby

### ISSUE THREAD

**Issue by ukabu** [mynyml/nanotest#1]
Hi,

in nanotest.rb, the split is not working when the file path contains colons (:) like when running on Windows.

Line 8 and 9 in the Nanotest class should be replaced with the following to make it work... With the added benefit of not doing the same split twice... and it does not increase the line counts ;-)

```
  groups = caller.first.match /(.*):(\d+)$/
  file, line = groups[1], groups[2]
```

**Comment by ukabu** [comment]
Sorry... You do have to increase the line count to make it possible to override the file and line...

```
  groups = caller.first.match /(.*):(\d+)$/
  file ||= groups[1]
  line ||= groups[2]
```

**Comment by mynyml** [comment]
Handle filenames with colons. Closed by 09ad388bb6a42990652b2e760eb204db2755d29a

Thanks to ukabu for the report and initial patch.

**Comment by mynyml** [comment]
Thanks for the report and initial patch, ukabu. Turns out it actually decreases the line count by one line. I released a new gem version with this fix.
